routes/discord.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+const authorizeBot = require("../middlewares/authorizeBot");
+const validator = require("../middlewares/validators/discord");
+const discord = require("../controllers/discord");
+
+router.post("/", validator.discordData, authorizeBot, discord.addDiscordData);
+
```

Comment:
add 1 more route to GET the id with the matching token

Reply:
Will raise a separate PR for that, this PR has already lot of changes.

Reply:
ok please create a ticket and link it here so that it is not forgotten

Reply:
Created a task in the same issue and updated the description
https://github.com/Real-Dev-Squad/website-backend/issues/912#issue-1557035449